### PR TITLE
Read binaries as bytes instead of string

### PR DESCRIPTION
**Pull request by Footmau**
I suggest to replace string return parameter of GetContent() with bytes array because it is much more common case since most of users store some binary files here.

### CODE REVIEW

**Comment by tobischo** [comment]
While I understand the idea behind it, I am not sure I want to introduce this breaking change.

We can introduce a `GetByteContent()` func and a `GetStringContent()` (or `GetContentAsBytes()`, `GetContentAsString()`) and label `GetContent()` as deprecated

**Comment by Footmau** [comment]
Sounds good, I'll update the changes.

File: binary.go [hunk 1]
```diff
@@ -65,8 +65,8 @@ func (bs *Binaries) Add(c []byte) *Binary {
 	return &(*bs)[len(*bs)-1]
 }
 
-// GetContent returns a string which is the plaintext content of a binary
-func (b Binary) GetContent() (string, error) {
+// GetContent returns a bytes array which is the content of a binary
```
Comment (tobischo) [inline binary.go:68]: Needs adaption for matching godoc standards

File: binary.go [hunk 2]
```diff
@@ -78,16 +78,32 @@ func (b Binary) GetContent() (string, error) {
...
-			return "", err
+			return nil, err
 		}
-		return string(bts), nil
+		return bts, nil
+	}
+	return decoded, nil
+}
+
+// GetContent returns a string which is the plaintext content of a binary
```
Comment (tobischo) [inline binary.go:93]: `GetContentString returns...`

**Review by tobischo — CHANGES_REQUESTED** [review]
Looks good. It would be nice if we could have godoc comments with the matching func names

File: binary.go (hunk 2 above)
Comment (Footmau) [reply binary.go:93]: Thanks, done.

**Review by tobischo — APPROVED** [review]

**Pull request merged**